Google.PowerShell/Logging/GcLogCmdlet.cs

```diff
@@ -640,4 +640,53 @@ protected override void ProcessRecord()
             WriteLogEntriesResponse response = request.Execute();
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Remove logs.
+    /// </para>
+    /// <para type="description">
+    /// Removes a log by its name
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcLog -LogName "test-log".</code>
```

Comment:
Remove the ending period, since that isn't valid PowerShell, right? Same in the example below.